### ISSUE TITLE
long lines in the top level of bullets are truncated before they appear on "Front" in Anki

### ISSUE THREAD

**Issue by EntropyOrSloth** [c-okelly/org_to_anki#78]
# Please follow the template below to report your issue.

- [https://pastebin.com/zA9iSkuL] Raw text 
- [none] Error report provided by Anki
- [MacOS X Catalina 10.15.5] Operatin system
- [html] Original file type

### Any general information

I imported an HTML bullet list from MS Word and imported using this add-on. There were items like this: "animate noun 2nd declension masculine nominative singular". On importing into Anki using this add-on, the "Fronts" were mostly truncated. For example, to "animate noun 2nd declension masculine", instead of the original "animate noun 2nd declension masculine nominative singular" line.

The only way I found to fix this is to edit each card in Anki, which defeats the purpose of this add-on.

These are the input files: [russian declensions - nominative, accusative, genitive.zip](https://github.com/c-okelly/org_to_anki/files/4925952/russian.declensions.-.nominative.accusative.genitive.zip)

These include both the original MS Word file as well as the HTML file created from it which was imported into Anki via this add-on.

### Raw text of the file you tried to upload

The PasteBin text from here can probably be used to recreate my file, however better would be to use the original input files given above: [PasteBin URL link](https://pastebin.com/zA9iSkuL).

### Error report from the popup

There were no errors reported by Anki.

### What is your operating system

MacOS X Catalina 10.15.5

### What was the original file type

MS Word *.docx file which was then saved as a *.html, the latter then being imported into Anki.

**Comment by EntropyOrSloth** [comment]
Yeah, sorry, I didn't realize this was on Github until the very end. I'll
delete my other messages if I can.

**Comment by EntropyOrSloth** [comment]
I have deleted my other comments elsewhere, at least for the time being.

I just downloaded LibreOffice 6.4.5, installed it on my MacBook Pro and
will try importing my docx file and creating a new HTML file using
LibreOffice to import into Anki to see if this problem will reproduce using
LibreOffice too.

**Comment by c-okelly** [comment]
Thanks for all the info. Currently looking into the issue though might be a day or two to get to the bottom of it

**Comment by c-okelly** [comment]
Hey,

Thanks for sending on all the information. Especially the actually `html` files.

The short and long of the issue is there Word just does not produce predictable HTML when you save a Word document. It appears the superscript symbols actually break the each line and the parser stop looking for information after this.

Unfortunately after having done a bit of work with the Word HTML files I have found they are actually too difficult to support consistently across systems.

To close this ticket I'm going to do the following:

- [ ] Remove all word references from the docs

- [ ] Add a word to LibreOffice migration docs

**TLDR**
**There is a simple migration path though the also seems to fix your error:**

0. Install LibreOffice (Free / OpenSource Word equvilant)
1. Open the file in LibreOffice
2. Save as a HTML file
3. Upload file to Anki

**Comment by c-okelly** [comment]
Let me know if that doesn't work!
